.buildkite/shared-pipeline-vars

```diff
@@ -4,9 +4,8 @@
 # to set up some variables that will be interpolated in the `.yml` pipeline before uploading it.
 
 # 🎗️ If you update the image to a newer Xcode version, don't forget to also update the badge in the README.md file accordingly for consistency
```

Comment:
You should probably update [this line of the `README.md` as well](https://github.com/Automattic/pocket-casts-ios/blob/trunk/README.md?plain=1#L15) (which seems hasn't been updated in a while, as it still says `15.4+` even while the project has been using `16.1+` for a while… so a bit overdue doc update 😅 )

Reply:
Good catch @AliSoftware ! Updated it in [567f2e7](https://github.com/Automattic/pocket-casts-ios/pull/2738/commits/567f2e76e71d127b84c95be02ab1c662342a5229)